### ISSUE TITLE
Bug: In the dev version using Docker I received in the mail link "http://localhost:64100/validateEmail.php?v="

### ISSUE THREAD

**Issue by anegoda1995** [RetroAchievements/RAWeb#851]
When I creating account I received in my mail message:
```
You or someone using your email address has attempted to sign up for an account at RetroAchievements.org

If this was you, please click the following link to confirm this email address and complete sign up:

        http://localhost:64100/validateEmail.php?v=

If this wasn't you, please ignore this email.

Thanks! And hope to see you on the forums!

-- Your friends at RetroAchievements.org
```

The problem is here **lib/database/user.php:58**
```
$query = "INSERT INTO EmailConfirmations VALUES( '$user', '$emailCookie', $expiry )";
// log_sql($query);
$dbResult = s_mysql_query($query);
```

**$dbResult** with **false** value, because:
```
retroachievements-web> INSERT INTO EmailConfirmations VALUES( 'anegoda1995', 'icc11A7ipR7rLd8E', '1637785697' )
[2021-11-17 22:29:15] [22001][1292] Data truncation: Incorrect date value: '1637785697' for column 'Expires' at row 1
```

This problem is not appear on production. What I need to do for properly set up dev environment?
I used docker-compose setup and using **master** branch

**Comment by luchaos** [comment]
Interesting 🤔  Yeah, certainly not a problem on production. 
It's very likely some mysql `SQL_MODE` flag that is missing locally which would allow to insert a UNIX timestamp into the `Expires` column of type `Date`.
I'll test it with [your PR](https://github.com/RetroAchievements/RAWeb/pull/852) and see if i can reproduce it.

**Comment by Tsearo** [comment]
Was this fixed at some point? I recently switched to a docker setup and the verification email
![image](https://user-images.githubusercontent.com/4047771/166084789-014ac7cc-9c5a-4209-9f89-25663406e136.png)

**Comment by luchaos** [comment]
> Was this fixed at some point?

The date format was indeed off. It did not fail on production because of its very lax `SQL_MODE` configuration, the timestamps were incorrect nonetheless because of the same error, storing `0000-00-00`.

Has been fixed in #911.
